setup.py

```diff
@@ -210,9 +210,8 @@ def run(self):
                                          SETUP_METADATA["packages"])
 
         if "z" not in self.libraries:
-            zcmd = ['bash', '-c', 'cd ' + ZLIBDIR + ' && ( test Makefile -nt'
-                    ' configure || bash ./configure --static ) && make -f '
-                    'Makefile.pic PIC']
+            zcmd = ['bash', '-c', 'cd ' + ZLIBDIR + ' &&  bash ./configure '
+                    '--static && make -f Makefile.pic PIC']
```

Comment:
This was giving me problems, probably should be removed. Or is there any problem in regenerating the Makefile?